src/hx/react.clj

```diff
@@ -59,14 +59,14 @@
     (if (and opts-map? (-> body first :wrap))
       (let [wrapped-name (symbol (str display-name "-hx-wrapped"))]
         `(do (def ~wrapped-name ~(fnc* wrapped-name props-bindings body))
-             (when js/goog.DEBUG
-               (set! (.-displayName ~wrapped-name) ~(str *ns* "/" display-name)))
+             (when goog/DEBUG
+               (goog.object/set ^js/React.Component ~wrapped-name "displayName" ~(str *ns* "/" display-name)))
```

Comment:
since we're using `goog.object/set`, I think we can remove the type hint.

Reply:
Yes, without the type hint I have no warnings for my advanced compilation